pkg/agent/plugin/nodeattestor/k8ssat/sat.go

```diff
@@ -81,6 +89,8 @@ func (p *AttestorPlugin) AidAttestation(stream nodeattestorv1.NodeAttestor_AidAt
 }
 
 func (p *AttestorPlugin) Configure(_ context.Context, req *configv1.ConfigureRequest) (resp *configv1.ConfigureResponse, err error) {
+	p.log.Warn(fmt.Sprintf("The %q node attestor plugin has been deprecated in favor of the \"k8s_psat\" plugin and will be removed in a future release", pluginName))
```

Comment:
```suggestion
	p.log.Warnf("The %q node attestor plugin has been deprecated in favor of the \"k8s_psat\" plugin and will be removed in a future release", pluginName)
```
nit: slightly simpler way to do the same thing

Reply:
Forgot this was the plugin logger using the hclog interface, so we can't do what I'm proposing here.